recipes/pymodsecurity/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pymodsecurity" %}
+{% set version = "0.0.1" %}
+{% set build_number = 0 %}
+{% set hash_value = "e9570494282b71172f909cb3ecfd4b7b66fa29a3141439192a48b40939893495" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_value }}
+
+build:
+  number: {{ build_number }}
+  skip: true  # [not linux]
+  script_env:
+    - CC
+    - CXX
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - modsecurity
+    - setuptools
+    - python >=3.5
+    - pybind11 >=2.2
+    - git
+  host:
+    - modsecurity
+    - setuptools
+    - python >=3.5
```

Comment:
I think instead of restricting the python version here it should be added to the `skip`, but I'm not 100% on this. 

@conda-forge/staged-recipes What is the rule about skipping python versions vs restricting python versions? I didn't see anything in the docs (I might add it after this gets clarified). We use a version restriction with `noarch: python`, but then recently I saw a PR where that was causing issues for Windows (not the case here).

Reply:
Found this in the docs: https://conda-forge.org/docs/meta.html#building-noarch-packages

Reply:
Your package is not ``noarch`` because you depends of the platform, I mean, you are compiling code. Because of that it is dependent of platform